libs/openFrameworks/math/ofQuaternion.h

```diff
@@ -199,21 +199,21 @@ ofQuaternion::ofQuaternion() {
 
 
 //----------------------------------------
-ostream& ofQuaternion::operator<<(ostream& os) {
-    os << _v.x << ", " << _v.y << ", " << _v.z << ", " << _v.w;
+ostream& operator<<(ostream& os, const ofQuaternion &q) {
```

Comment:
i see. then it might work if the method implementation is moved from the .h file to the .cpp file.